### DISCUSSION TITLE
Hey please add

### DISCUSSION THREAD

**Discussion by SH8GH** [timlrx/tailwind-nextjs-starter-blog#174] [Ideas]
Hey repo's owner only degit, you should add command like 

```bash
npx create-next-app -e [this repo]
```

**Comment by timlrx** [comment]
`create-next-app` is a cli maintained by the nextjs team. Creating something similar would involve quite a bit of tooling and maintenance. `degit` is much simpler since it is literally just a copy of the repository / branch, without the entire git history.
